crates/common/src/selectors.rs

```diff
@@ -299,18 +299,25 @@ impl SignEthClient {
 
         let request = match data {
             SelectorImportData::Abi(abis) => {
-                let names: Vec<String> = abis
+                let functions_and_errors: Vec<String> = abis
                     .iter()
                     .flat_map(|abi| {
                         abi.abi
                             .functions()
                             .map(|func| {
                                 func.signature().split(':').next().unwrap_or("").to_string()
                             })
+                            .chain(abi.abi.errors().map(|error| error.signature()))
```

Comment:
I don't think this split is needed anymore, right @Evalir ?

Reply:
Yep I think it might not be needed anymore—now the actual abi comes from alloy